templates/about.hbs

```diff
@@ -8,27 +8,28 @@
 
   <p>
   Docs.rs automatically checks crates released into
-  <a href="https://crates.io/">crates.io</a> and builds documentations. It is
-  using nightly release of Rust compiler.
+  <a href="https://crates.io/">crates.io</a> and builds documentations. It
+  uses nightly release of Rust compiler.
   </p>
 
   <p>
-  The source code for Docs.rs is available
-    <a href="https://github.com/onur/docs.rs" target="_blank">on Github</a>. If
-  you encountered any issue, feel free to report.
+  The source code of Docs.rs is available on
+    <a href="https://github.com/onur/docs.rs" target="_blank">Github</a>. If
+  you ever come to encounter an issue, don't hesitate to report it! (And
+  thanks in advance!)
   </p>
 
   <p>
   README of a crate is taken from <code>readme</code> field defined in
-  <code>Cargo.toml</code>. If a crate doesn't have a readme field in
-  <code>Cargo.toml</code>, it will not display any README for this crate.
+  <code>Cargo.toml</code>. If a crate doesn't have this field,
+  it won't display any README for this crate.
```

Comment:
a README won't be displayed

Shorter, and doesn't repeat crate, but I'm not 100% about it